sdk/storage/storage-blob/src/generated/src/models/parameters.ts

```diff
@@ -102,7 +102,7 @@ export const timeoutInSeconds: OperationQueryParameter = {
 export const version: OperationParameter = {
   parameterPath: "version",
   mapper: {
-    defaultValue: "2021-12-02",
+    defaultValue: "2023-01-03",
```

Comment:
do we also have to update the service version in `utils/constants.ts`?

Reply:
I think so. I missed it.